dev/src/DocFx/Node/FencedCodeBlockTrait.php

```diff
@@ -29,9 +29,12 @@ trait FencedCodeBlockTrait
     private function addPhpLanguageHintToFencedCodeBlock(string $description): string
     {
         return preg_replace_callback(
-            '/^(\s+)?```\n((.|\n)*)\n^(\s+)?```$/m',
+            '/^(\s+)?```(\w+)?\n((.|\n)*)\n^(\s+)?```$/mU',
```

Comment:
the new `(\w+)?` matches a potential existing language hint, and the `U` modifier makes the regex ungreedy.